packages/hoppscotch-backend/package.json

```diff
@@ -94,8 +94,8 @@
     "@typescript-eslint/eslint-plugin": "8.8.1",
     "@typescript-eslint/parser": "8.8.1",
     "cross-env": "7.0.3",
-    "eslint": "9.12.0",
-    "eslint-config-prettier": "9.1.0",
+    "eslint": "8.57.0",
+    "eslint-config-prettier": "8.10.0",
```

Comment:
I think we can still be on `eslint-config-prettier` `v9` being backwards compatible with ESLint `v8`, `peerDependencies` refers to ESLint v7+, [ref](https://github.com/prettier/eslint-config-prettier/blob/main/package.json#L16-L18).